.travis.yml

```diff
@@ -33,6 +33,7 @@ env:
     - MODE=aot
     - MODE=payload
     - MODE=e2e
+    - MODE=docs
```

Comment:
The whole mode shouldn't be necessary when we run it with the `travis-after-modes` script.